MobileCryptoService/MobileCryptoService/Presentation/HomeScreen/View/ProfileView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ProfileView.swift
+//  MobileCryptoService
+//
+//  Created by Dario Leonel Abatte on 07/11/2022.
+//
+
+import SwiftUI
+
+struct ProfileView: View {
+    var body: some View {
+        ZStack{
+            Color("BackgroundColor")
+                .edgesIgnoringSafeArea(.all)
+            VStack{
+
+                Image("ProfilePhoto")
+                    .padding()
+                Text("Jose Perez")
```

Comment:
@LeonelAbatteEndava  I will create a little technical task for create a  font style provider :) in order to reuse the style and define ones